src/commands/issue.ts

```diff
@@ -0,0 +1,25 @@
+import { IMigrationContext } from '../migration-context';
+import forEachRepo from '../util/for-each-repo';
+
+export default async (context: IMigrationContext) => {
+  const {
+    migration: { spec },
+    logger,
+  } = context;
+
+  if (!spec.hooks.issue_message || !spec.hooks.issue_labels) {
```

Comment:
What's the rational behind making labels required?

Reply:
Should I make both issue message and labels as optional

Reply:
It depends on the rational for making them required.  Sometimes Issues are opened with just a title; I can see title being required.

Reply:
yes i made both issue message and labels as optional